src/field/social/social_buttons_pane.jsx

```diff
@@ -6,6 +6,16 @@ import { logIn } from '../../quick-auth/actions';
 import { displayName, socialConnections, authButtonsTheme } from '../../connection/social/index';
 
 export default class SocialButtonsPane extends React.Component {
+  handleAuthButtonClick(lock, provider, isSignUp) {
+    const prov = typeof provider !== 'object' ? provider.toJS() : provider;
```

Comment:
Its technically possible to have multiple strategies of the same type though, right? Thats why we were thinking of getting the `name` as well as the `strategy`. We dont need all the properties though.